test/cases/ostree-simplified-installer.sh

```diff
@@ -102,24 +102,29 @@ SSH_KEY_PUB=$(cat "${SSH_KEY}".pub)
 # kernel-rt package name (differs in CS8)
 KERNEL_RT_PKG="kernel-rt"
 
+# Only RHEL 9.2 and above support user customizations in simplified installer blueprint
 case "${ID}-${VERSION_ID}" in
     "rhel-8.8")
         OSTREE_REF="rhel/8/${ARCH}/edge"
         OS_VARIANT="rhel8-unknown"
+        USER_IN_SIMPLIFIED="false"
         ;;
     "rhel-9.2")
         OSTREE_REF="rhel/9/${ARCH}/edge"
         OS_VARIANT="rhel9-unknown"
+        USER_IN_SIMPLIFIED="true"
         ;;
     "centos-8")
         OSTREE_REF="centos/8/${ARCH}/edge"
         OS_VARIANT="centos8"
         KERNEL_RT_PKG="kernel-rt-core"
+        USER_IN_SIMPLIFIED="false"
```

Comment:
just note that if this change will get into centos-8 as well, so it should be possible to test it there if it is desired.

Reply:
> @henrywang let me know if the updated tests make sense to you, if so, feel free to squash the commit with your original one

Thanks! Commit squashed.